pkg/ovs/openflow/interfaces.go

```diff
@@ -166,6 +184,19 @@ type FlowBuilder interface {
 	Done() Flow
 }
 
+type Group interface {
+	Entry
+	Bucket() BucketBuilder
+}
+
+type BucketBuilder interface {
+	Weight(val uint16) BucketBuilder
+	LoadReg(regID int, data uint32) BucketBuilder
```

Comment:
I'm curious as to why we are limited to `LoadReg` / `LoadRegRange` / `ResubmitToTable` here? Shouldn't we be using `Action()` (same as for flows)?

Reply:
I see. The issue is that the methods in the `Action` interface return a `FlowBuilder` object. Maybe we can refactor the code later if we need to support more actions for buckets.

Reply:
Yes, the ActionBuilder is coupled with FlowBuilder for now, we may need to abstract an ActionApplier later.